### DISCUSSION TITLE
Binding  snake_case property doesn't work.

### DISCUSSION THREAD

**Discussion by levipeto** [livewire/livewire#4389] [1. Help] [answered]
Hello,

I have a project that uses Laravel Spark that stores billing info as (for example) "billing_address" in the users' table.
Binding to a snake case property doesn't work for me. 

**Syntax that works:**
wire:model="billing_address"
wire:model="user.name"

**Syntax, that doesn't Work:**
wire:model="user.billing_address"

I am using  Livewire v2.8.1 

I tried to google it, but I didn't find any solution. What am I missing?

Thanks for the help!

**Comment by levipeto** [comment, marked as answer]
Please delete this thread. Thank you!

**Reply by joshhanley** [reply]
@levipeto did you manage to get it working? If so can you share incase others come across the same issue? Thanks! 🙂

**Reply by levipeto** [reply]
Yes, I did. The issue was that I had to set up rules() for validating the new fields.

**Reply by joshhanley** [reply]
Ah yep, no worries, thanks for sharing 🙂
